*** core_cmInstr.h.orig	2011-10-03 19:48:04.000000000 +0900
--- core_cmInstr.h	2012-02-07 23:52:08.000000000 +0900
***************
*** 465,471 ****
  {
     uint32_t result;
    
!    __ASM volatile ("strexb %0, %2, [%1]" : "=r" (result) : "r" (addr), "r" (value) );
     return(result);
  }
  
--- 465,471 ----
  {
     uint32_t result;
    
!    __ASM volatile ("strexb %0, %2, [%1]" : "=&r" (result) : "r" (addr), "r" (value) );
     return(result);
  }
  
***************
*** 483,489 ****
  {
     uint32_t result;
    
!    __ASM volatile ("strexh %0, %2, [%1]" : "=r" (result) : "r" (addr), "r" (value) );
     return(result);
  }
  
--- 483,489 ----
  {
     uint32_t result;
    
!    __ASM volatile ("strexh %0, %2, [%1]" : "=&r" (result) : "r" (addr), "r" (value) );
     return(result);
  }
  
***************
*** 501,507 ****
  {
     uint32_t result;
    
!    __ASM volatile ("strex %0, %2, [%1]" : "=r" (result) : "r" (addr), "r" (value) );
     return(result);
  }
  
--- 501,507 ----
  {
     uint32_t result;
    
!    __ASM volatile ("strex %0, %2, [%1]" : "=&r" (result) : "r" (addr), "r" (value) );
     return(result);
  }
  
